mappings/net/minecraft/command/CommandLocateStructure.mapping

```diff
@@ -1,7 +1,14 @@
-CLASS ax
+CLASS bw net/minecraft/command/CommandLocateStructure
```

Comment:
Matching the command name is probably good, but since it includes "Locate" as the first term, it'll still be found by people assuming names by the command's name. This name is more descriptive.

Mods may add their own non-structure markers to this, however.